tests/test_send_recv_two_workers.py

```diff
@@ -191,6 +191,9 @@ def cupy_obj():
 )
 @pytest.mark.parametrize("comm_api", ["tag", "am"])
 def test_send_recv_cu(cuda_obj_generator, comm_api):
+    if comm_api == "am" and not ucp._libs.ucx_api.is_am_supported():
+        pytest.skip("AM only supported in UCX >= 1.11")
+
```

Comment:
This change is unrelated to the PR's description, I intended to open a separate PR but accidentally had it pushed here. It should be harmless but necessary, so I'll just leave it here anyway.